examples/index.html

```diff
@@ -17,12 +17,10 @@
       padding: 2rem;
       font-size: 3.6rem;
       border-radius: 1rem;
-      position: absolute;
-      top: 1rem;
-      left: 1rem;
     }
 
   </style>
+  <link rel="stylesheet" type="text/css" href="../bower_components/smart-icons/smart-icons.css">
```

Comment:
We should add `"smart-icons": "smart-components/smart-icons"` to dependencies of bower.json.